pallets/nfts/src/features/approvals.rs

```diff
@@ -76,12 +77,11 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 
 		Self::deposit_event(Event::TransferApproved {
 			collection,
-			item,
+			item: Some(item),
```

Comment:
Note: Tested in:

- approving_multiple_works: https://github.com/r0gue-io/pop-node/pull/387/files#r1881289915